test/dotnet-new3.UnitTests/DotnetNewInstall.cs

```diff
@@ -383,7 +383,10 @@ public async Task InstallingSamePackageFromRemoteUpdatesLocal()
             var home = TestUtils.CreateTemporaryFolder("Home");
 
             using var packageManager = new PackageManager();
-            string packageLocation = await packageManager.GetNuGetPackage("Microsoft.DotNet.Common.ProjectTemplates.5.0", _log).ConfigureAwait(false);
+            string packageLocation = await packageManager.GetNuGetPackage(
+                "Microsoft.DotNet.Common.ProjectTemplates.5.0",
+                minimumVersion: new NuGet.Versioning.NuGetVersion(6, 0, 0),
```

Comment:
Can we tie this to the current assembly version? Or the version in "Versions.xml" file somehow?

Reply:
for the test purpose it's not required to be an actual SDK version, it's just need to have different majors.
3.1 and 5.0 will work as well, for example.